### PR TITLE
jupyter example notebook added

### PR DESCRIPTION
**Pull request by sroet**

### CODE REVIEW

**Comment by sroet** [comment]
@andersle this is ready for review and merge

File: exercises/00_jupyter_introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving your work\n",
+    "\n",
+    "Due to hosting this through Binder, any changes that you make to the notebooks or filesystem will **not** be stored when you close all the browser windows or press the *Quit* button on the file selection screen. \n",
+    "\n",
+    "If you want to keep your answers and alterations of the notebook you are working in, please click *File* -> *Download as* -> *Your prefered file format*. "
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "If you want to keep your answers and alterations of the notebook you are working in, please click *File* -> *Download as* -> *Your preferred file format*. "
```

File: exercises/00_jupyter_introduction.ipynb [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Making plots\n",
+    "\n",
+    "Plots are a nice way of visualizing data. In python we use a package called `matplotlib` for that. and for fast math and array operations we will use `numpy`. We will now show an example on how to plot the cosine of 360 degrees.\n",
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "Plots are a nice way of visualizing data. In Python we use a package called `matplotlib` for that. and for fast math and array operations we will use `numpy`. We will now show an example on how to plot the cosine of 360 degrees.\n",
```

File: exercises/00_jupyter_introduction.ipynb [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    "\n",
+    "First we will import the packages we will be using"
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "First we will import the packages we will be using:"
```

File: exercises/00_jupyter_introduction.ipynb [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+   "source": [
+    "**Note:**\n",
+    "If you want to know more about a function that we are using, like `numpy.arrange()`, you can click to place the cursor inside the function parentheses and press \"Shift+Tab\" (press more than once for more info)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we calculate the cosine and plot it"
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "Now we calculate the sine and plot it:"
```

File: exercises/00_jupyter_introduction.ipynb [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = np.sin(rad)\n",
+    "plt.plot(rad, y, label='sin') # Plot x, y\n",
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "fig, ax = plt.subplots(nrows=1, ncols=1)  # Create a figure and axes\n",
```

File: exercises/00_jupyter_introduction.ipynb [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    "plt.legend()  # Plot the legends\n",
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "ax.plot(rad, y, label='sin') # Plot x, y\n",
```

File: exercises/00_jupyter_introduction.ipynb [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+    "plt.xlabel('x')  # Set the label for the x-axis\n",
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "ax.legend()  # Plot the legends\n",
```

File: exercises/00_jupyter_introduction.ipynb [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+    "plt.ylabel('y')  # Set the label for the y-axis"
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "ax.set(xlabel='x', ylabel='y')  # Set the label for the x-axis and y-axis"
```

File: exercises/00_jupyter_introduction.ipynb [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Saving plots\n",
+    "In order to save the **latest** plot as a .png you can run `plt.savefig('name.png')`.\n",
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "In order to save this plot as a .png you can run `fig.savefig('name.png')`.\n",
```

File: exercises/00_jupyter_introduction.ipynb [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+    "It will then show up on the file selection screen, where you can select it and press the *Download* button to download it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.savefig('test.png')"
```
Comment (andersle) [inline exercises/00_jupyter_introduction.ipynb]:
```suggestion
    "fig.savefig('test.png')"
```

**Comment by sroet** [comment]
@andersle applied sugestion, please have another look

**Pull request merged**